File: H5/H5/Resources/.generated/System/Diagnostics/Debugger.js
    H5.define("System.Diagnostics.Debugger", {
        statics: {
            fields: {
                DefaultCategory: null
            },
            props: {
                IsAttached: {
                    get: function () {
                        return true;
                    }
                }
            },
            methods: {
                IsLogging: function () {
                    return true;
                },
                Launch: function () {
                    return true;
                },
                Log: function (level, category, message) { },
                NotifyOfCrossThreadDependency: function () { }
            }
        }
    });

File: H5/H5/runtime/Debugger.cs
namespace System.Diagnostics
{
    [H5.Reflectable(false)]
    public static class Debugger
    {
        [H5.Template("debugger")]
        public static extern void Break();

        public static readonly string DefaultCategory;
        public static bool IsAttached { get { return true; } }
        public static bool IsLogging() { return true; }
        public static bool Launch() { return true; }
        public static void Log(int level, string category, string message) { }
        public static void NotifyOfCrossThreadDependency() { }
    }
}